src/makefun/main.py

```diff
@@ -255,6 +294,8 @@ def create_function(func_signature,             # type: Union[str, Signature]
             body = get_legacy_py_generator_body_template() % (func_signature_str, params_str)
     elif isasyncgenfunction(func_impl):
         body = "async def %s\n    async for y in _func_impl_(%s):\n        yield y\n" % (func_signature_str, params_str)
+    elif create_lambda:
+        body = "lambda_ = lambda%s: _func_impl_(%s)\n" % (func_signature_str, params_str)
```

Comment:
This could probably be the place where to add the extra space ? (see previous comment)